server.ts

```diff
@@ -27,10 +29,34 @@ const client = new Client(dbConfig);
 client.connect();
 
 app.get("/", async (req, res) => {
-  const dbres = await client.query('select * from categories');
+  const dbres = await client.query('select * from pastedata limit 10');
   res.json(dbres.rows);
+  
 });
 
+app.post("/", async (req,res)=> {
```

Comment:
No types here to let TS know what the body should like. Potentially problematic because nothing is being returned to the client if
`input.title` 
throws an error. The client could be hanging indefinitely and the user will be none the wiser.